vint.go

```diff
@@ -65,8 +65,14 @@ func mustReadByte(r io.Reader) (byte, error) {
 // implemented here: https://goo.gl/1h4mrG. It does at most one write to the
 // underlying io.Writer
 func WriteVInt(w io.Writer, n int64) (int, error) {
+	return w.Write(PutVInt(n))
+}
+
+// PutVInt returns a byte slice containing an int64 encoded in hadoop's "VInt" format,
+// described and implemented here: https://goo.gl/1h4mrG
+func PutVInt(n int64) []byte {
```

Comment:
is there a way to make it less odd that you could see being exported?

the vint stuff I can kinda see as eventually getting extracted out as a generic library, along with *writable and such. Like a "hadoop data format library" ... which would also mirror the java implementation that does the same thing. I think in that case PutVint would have to be exported for sequencefile to be able to use it, yes?